keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -234,11 +226,11 @@ A `Bucket` is not deleted if it is bound to a `BucketRequest`.
 1. `allowedNamespaces`: a list of namespaces that are permitted to either create new buckets or to access existing buckets.
 1. `protocol`: protocol-specific field the application will use to access the backend storage.
      - `name`: supported protocols are: "s3", "gs", and "azureBlob".
-     - `version`: version being used by the backend storage.
-     - `azureBlob`: data required to target a provisioned azure container.
-     - `gs`: data required to target a provisioned GS bucket.
-     - `s3`: data required to target a provisioned S3 bucket.
-1. `parameters`: a copy of the BucketClass parameters.
+     - `version`: protocol version being used by the backend storage.
+     - `bucketName`: the name of the bucket created by the backend storage.
+     - `endpoint`: the url to the bucket endpoint.
+     - `parameters`: a map of string:string copied from the BucketClass parameters.
+1. `parameters`: a map of string:string copy of the BucketRequest fields related to bucket naming.
```

Comment:
Do `parameters` here also contain BucketClass.parameters?

Reply:
They aren't merged.  b.protocol.parameters is a copy of bc.parameters.  b.parameters is fields from br like BucketInstanceName and BucketPrefix, as those are what is needed to get a bucket name to/from the driver.  It is the driver's responsibility to create a bucket name and return it if one isn't defined already.

Reply:
Ah, I missed seeing 2 `parameters` in the `Bucket` definition.